src/main.ts

```diff
@@ -429,12 +457,40 @@ async function pkgid(crate: CrateArgs = {}): Promise<CrateDetails> {
 				(crate.name && crate.name === parsed.name) ||
 				(cratePath && cratePath === parsed.path)
 			)
-				return parsed;
+				return [parsed];
 		}
 	} else if (pkgs.length === 1) {
 		info('only one crate in workspace, assuming that is it');
 		const parsed = parseWorkspacePkg(pkgs[0]);
 		if (!parsed) throw new Error('no good crate found');
+		return [parsed];
+	} else {
+		info(
+			'multiple crates in the workspace, releasing all if crate-release-all option is set'
+		);
+		if (!crate.releaseAll) throw new Error('release-all option not set');
```

Comment:
I'd prefer to avoid the conditional in the log, and instead do something like

- if releaseAll
  - info(multiple crates in workspace, releasing all)
- else
  - error(multiple crates in workspace but crate-release-all=false)
  - throw